### ISSUE TITLE
[REQ]  capture app name + window title of clip in desc. window

### ISSUE THREAD

**Issue by Rainroom69** [sabrogden/Ditto#174]
Hello Scott,
does it make sense to capture both Application name (example notepad.exe) and the window title (ex. Main goals) in the clip description box ?  Why this request ?
I experience to have a lot of clips in the main list (not in groups) and often see/need a clip with text which I cannot remember from WHERE I collected the clip...    Having both app name and window title captured helps me to remind from where I got it,  making it often easier to go back to that app/window.   Having the option to enable this 'feature' , each user can decide to activate this or not...   for programmers also easy to remember where the clip came from ... 
 Thanks to consider if worth to have or not.  For me it make sense ;-)

**Comment by peter289** [comment]
I would also find this helpful

**Comment by Chealer** [comment]
I agree this would be an improvement. 03.25.113 cannot do that. I do not know if the Windows API allows doing that, but the fact that Microsoft OneNote allows pasting the source URL (as discussed in ticket #482) suggests it is.

Since this is an ITS and I very much hope this could be used for more than just display (at least filtering, and the above ticket even suggests a special paste), I suggest retitling to something like:
> Clip's source application name and window title is not captured (for display/filtering)

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>
